.gitignore

```diff
@@ -56,4 +56,5 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
\ No newline at end of file
+.env
+.env.example

```

Comment:
Currently, the [docker file](https://github.com/RollInitiative/DiscordBot/blob/99691eecffe081105089ad0f105e612e6ae8863e/Dockerfile#L10) copies the example file as `.env`. So yeah, these two files shouldn't be ignored for now.